specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,103 @@
+|                           | Data Warehouse                    | Centralized repository that stores large volumes of structured data to facilitate analytical use cases.                                                                                                |
+|                           | Relational                        | Data organized into tables with rows and columns to facilitate transactional use cases.                                                                                                                |
+|                           | NoSQL                             | Unstructured or semi-structured data to facilitate use cases where data models change frequently.                                                                                                      |
+|                           | Ledger                            | Transparent, immutable, and cryptographically verifiable transaction log used in applications requiring a secure and auditable record of transactions.                                                 |
+|                           | Time Series                       | Data store optimized for storing and querying time-stamped data to facilitate use cases where tracking data changes over time is crucial.                                                              |
+|                           | Other (Databases)                 | Database services that do not fall into one of the defined subcategories.                                                                                                                              |
+| Developer Tools           | Other (Developer Tools)           | All Developer Tools services.                                                                                                                                                                          |
+| Identity                  | Other (Identity)                  | All Identity services.                                                                                                                                                                                 |
+| Integration               | API Management                    | Creation, publishing, and governance of application programming interfaces that facilitate communication between software systems.                                                                     |
+|                           | Messaging                         | Asynchronous communication between distributed applications.                                                                                                                                           |
+|                           | Workflow Automation               | Use of software to complete orchestration tasks and activities without the need for human input.                                                                                                       |
```

Comment:
```suggestion
|                           | Workflow Automation               | Design, execution, and management of business processes and workflows.                                                                                                       |
```

Reply:
I was really trying to avoid the use of the term `workflow`.  Could we go with:

```suggestion
|                           | Workflow Automation               | Design, execution, and management of repeatable business processes.                                                                                                       |
```